node/src/chain_spec/flashbox.rs

```diff
@@ -0,0 +1,319 @@
+    crate::chain_spec::{
+        account_ids, get_account_id_from_seed, invulnerables_from_seeds, Extensions,
+    },
+    cumulus_primitives_core::ParaId,
+    flashbox_runtime::{
+        prod_or_fast, AccountId, DataPreserversConfig, MaintenanceModeConfig, MigrationsConfig,
+        RegistrarConfig, ServicesPaymentConfig, SudoConfig,
+    },
+    nimbus_primitives::NimbusId,
+    pallet_configuration::HostConfiguration,
+    sc_service::ChainType,
+    sp_core::sr25519,
+    sp_runtime::traits::Get,
+    tp_container_chain_genesis_data::{
+        json::container_chain_genesis_data_from_path, ContainerChainGenesisData,
+    },
+};
+
+/// Specialized `ChainSpec` for the normal parachain runtime.
+pub type ChainSpec =
+    sc_service::GenericChainSpec<flashbox_runtime::RuntimeGenesisConfig, Extensions>;
+
+/// Generate the session keys from individual elements.
+///
+/// The input must be a tuple of individual keys (a single arg for now since we have just one key).
+pub fn template_session_keys(keys: NimbusId) -> flashbox_runtime::SessionKeys {
+    flashbox_runtime::SessionKeys { nimbus: keys }
+}
+
+pub fn development_config(
+    para_id: ParaId,
+    container_chains: Vec<String>,
+    mock_container_chains: Vec<ParaId>,
+    invulnerables: Vec<String>,
+) -> ChainSpec {
+    // Give your base currency a unit name and decimal places
+    let mut properties = sc_chain_spec::Properties::new();
+    properties.insert("tokenSymbol".into(), "FLASH".into());
+    properties.insert("tokenDecimals".into(), 12.into());
+    properties.insert("ss58Format".into(), 42.into());
+    properties.insert("isEthereum".into(), false.into());
+
+    ChainSpec::from_genesis(
+        // Name
+        "Flashbox Development Testnet",
+        // ID
+        "flashbox_dev",
+        ChainType::Development,
+        move || {
+            testnet_genesis(
+                // initial collators.
+                invulnerables_from_seeds(invulnerables.iter()),
+                account_ids(&[
+                    "Alice",
+                    "Bob",
+                    "Charlie",
+                    "Dave",
+                    "Eve",
+                    "Ferdie",
+                    "Alice//stash",
+                    "Bob//stash",
+                    "Charlie//stash",
+                    "Dave//stash",
+                    "Eve//stash",
+                    "Ferdie//stash",
+                ]),
+                para_id,
+                get_account_id_from_seed::<sr25519::Public>("Alice"),
+                &container_chains,
+                &mock_container_chains,
+                pallet_configuration::GenesisConfig {
+                    config: HostConfiguration {
+                        max_collators: 100u32,
+                        min_orchestrator_collators: 1u32,
+                        max_orchestrator_collators: 1u32,
+                        collators_per_container: 2u32,
+                        full_rotation_period: prod_or_fast!(24u32, 5u32),
```

Comment:
This field is ignored in flashbox, not sure if there is an easy way to remove it here but keep it in dancebox. Anyway 0 means no rotation so it's a better value.
```suggestion
                        full_rotation_period: 0,
```